response_operations_ui/views/messages.py

```diff
@@ -161,6 +161,10 @@ def _view_select_survey(marked_unread_message=""):
                             flash_message=marked_unread_message))
 
 
+def remove_whitespace_from_survey_name(survey):
```

Comment:
Wondering if this should be moved somewhere else like into the common folder?

Reply:
Agreed.  If this was only being used in the one view, it'd be fine.  But a view shouldn't be importing functions from another view.